settings.gradle

```diff
@@ -105,6 +105,10 @@ if (!System.getenv().containsKey("SUB_BUILD") || System.getenv().get("SUB_BUILD"
     include ':tools:code-generator'
 }
 
+if (!System.getenv().containsKey("SUB_BUILD") || System.getenv().get("SUB_BUILD") == "OCTAVIA_CLI") {
```

Comment:
would it make sense to just have it be "CLI" instead of "OCTAVIA_CLI"? just easier to type.

Reply:
I like having a shorter name. Would this be confused with our Go cli?

Reply:
My opinion is that the `OCTAVIA` prefix allows to disambiguate from the `airbyte-cli` task already declared in this file.